feature-flags.md

```diff
@@ -0,0 +1,42 @@
+## `data_loss_protect`
+
+Accepted in November 2017: [meeting log](https://docs.google.com/document/d/1x2Qr2VgqauFfPgs-uBv7B4GegifAzRmfA_LpebvsETM/edit)
+
+This option is required by most implementations, e.g. LND: [brontide.go:2749](https://github.com/lightningnetwork/lnd/blob/cc0321d1881ed23c9608cf898af2e2a7b347304a/peer/brontide.go#L2749).
+
+## `static_remote_key`
+
+This removes key-rotation from the `to_remote` output. E.g. watchtowers are easier to implement this way.
+
+### Implementations
+
+- C-Lightning: since [this PR from September 2019](https://github.com/ElementsProject/lightning/pull/3104), part of 0.7.3 according to the milestone there.
+- Eclair: since [this PR merged in June 2020](https://github.com/ACINQ/eclair/pull/1141), part of v0.5.0 according to [branch information](https://github.com/ACINQ/eclair/commit/dc364a1996548e405199fe38ad861a693baf4787)
+- LND: since [v0.8.0 from October 2019](https://github.com/lightningnetwork/lnd/releases/tag/v0.8.0-beta). LND requires this option for new channels: [brontide.go:2790](https://github.com/lightningnetwork/lnd/blob/cc0321d1881ed23c9608cf898af2e2a7b347304a/peer/brontide.go#L2790)
+- Electrum requires this for all peers since [4.0.1 from July 2020](https://github.com/spesmilo/electrum/blob/922a48f2b74bd6c58682f9d5b6163c4fbd45981a/RELEASE-NOTES#L107)
+
+## `payment_secret`
+
+Merged in July 2019 as part of MPP: https://github.com/lightningnetwork/lightning-rfc/pull/643
+
+The feature prevents probing attacks as noted in BOLT-11.
+
+You can use payment_secret without necessarily using MPP, though it is a requirement for MPP.
+
+### Implementations
+
+- C-Lightning: since at least [v0.8.0 from December 2019](https://github.com/ElementsProject/lightning/releases/tag/v0.8.0), since that version supports receiving MPP
+- Eclair: since at least [this PR from October 2019](https://github.com/ACINQ/eclair/pull/1153), released with MPP (see below)
+- Electrum: sent/accepted since [4.0.1 from July 2020](https://github.com/spesmilo/electrum/blob/922a48f2b74bd6c58682f9d5b6163c4fbd45981a/RELEASE-NOTES#L107)
+- LND: supported since v0.10.0 (see below), **required** in incoming payments since [v0.12.0](https://github.com/lightningnetwork/lnd/pull/4752) (unreleased as of January 7th, 2020):  All generated invoices will include the `s` field.
+
+## Multi-part payments (MPP)
+
+Contrast with e.g. "High" Atomic Multi-part Payments (AMP), which is different, not yet standardized, but mentioned in BOLT-04.
+
+### Implementations
+All listed implementations that have send support, also have receive support.
+
+- C-Lightning: Send support since [v0.9.0 from July 2020](https://github.com/ElementsProject/lightning/releases/tag/v0.9.0)
+- Eclair: Send support since [v0.3.3 from January 2020](https://github.com/ACINQ/eclair/releases/tag/v0.3.3), see also [wiki page](https://github.com/ACINQ/eclair/wiki/Multipart-Payments). Needs manual enabling.
```

Comment:
```suggestion
- Eclair: Send support since [v0.3.3 from January 2020](https://github.com/ACINQ/eclair/releases/tag/v0.3.3), see also [wiki page](https://github.com/ACINQ/eclair/wiki/Multipart-Payments).
```

I've updated the wiki page, we turned this on by default a while ago (for both sending and receiving).